app/scripts/directives/processTemplate.js

```diff
@@ -152,16 +153,32 @@ function ProcessTemplate($filter,
     modalInstance.result.then(createResources);
   };
 
+  var alerts = {};
+  var hideNotificationErrors = function() {
+    NotificationsService.hideNotification("process-template-error");
+    _.each(alerts, function(alert) {
+      if (alert.id) {
```

Comment:
this assumes that we will only ever put id's on the error alerts, should this also check that the alert is a warning or error alert